arch/X86/X86GenRegisterInfo.inc

```diff
@@ -382,10 +383,11 @@ enum {
   X86_VR512_with_sub_xmm_in_VR128HRegClassID = 84,
   X86_VR512_with_sub_xmm_in_VR128LRegClassID = 85,
 };
+
 #endif // GET_REGINFO_ENUM
 
 #ifdef GET_REGINFO_MC_DESC
-#define GET_REGINFO_MC_DESC
+#undef GET_REGINFO_MC_DESC
```

Comment:
check git blame to see if this was fixings something

Reply:
I found specific commits regarding GET_REGINFO_MC_DESC
```
0b690387b (Nguyen Anh Quynh  2014-08-13 13:01:50 +0800  387) #ifdef GET_REGINFO_MC_DESC
5a9962407 (Nguyen Anh Quynh  2019-02-26 15:19:51 +0800  388) #define GET_REGINFO_MC_DESC
0b690387b (Nguyen Anh Quynh  2014-08-13 13:01:50 +0800 1549) #endif // GET_REGINFO_MC_DESC
```
It looks like the second line is hand-edited.
So, I will make the change by hand.